include/range/v3/utility/common_tuple.hpp

```diff
@@ -415,42 +415,42 @@ namespace ranges
                     std::tuple<Ts...>>;
         }
 
-        // common_type for std::pairs
+        // common_type for pairs
         template<typename F1, typename S1, typename F2, typename S2>
         struct common_type<std::pair<F1, S1>, common_pair<F2, S2>>
           : detail::common_type_tuple_like<std::pair<F1, S1>, common_pair<F2, S2>,
-                meta::quote<std::pair>>
+                meta::quote<detail::make_common_pair>>
```

Comment:
>  I also vaguely recall there being a reason to favor the `std` flavors

It's possible it was simply an attempt to minimize the disruption of introducing the `common_foo` types. Anyway, the tests don't seem to care.